src/create-key.ts

```diff
@@ -14,12 +13,24 @@ const deepSortObject = (opts?: any): any => {
     }, {});
 };
 
+function padStart(hash: string, len: number) {
+  while (hash.length < len) {
+    hash = '0' + hash;
+  }
+  return hash;
+}
```

Comment:
After having tested a few algorithms, I think that the Ivan Perelivskiy's TinySimpleHash, [refered here](https://gist.github.com/iperelivskiy/4110988?permalink_comment_id=3716618#gistcomment-3716618) does an excellent job and is strong enough for this use-case.
More info can be found [here](https://stackoverflow.com/a/52171480/815680)